packages/client/index.js

```diff
@@ -93,7 +94,12 @@ function hasDuplicatedParameters (methodMeta) {
   })
   return s.size !== methodMeta.parameters.length
 }
-function buildCallFunction (baseUrl, path, method, methodMeta, throwOnError, openTelemetry, fullRequest, fullResponse) {
+function buildCallFunction (spec, baseUrl, path, method, methodMeta, throwOnError, openTelemetry, fullRequest, fullResponse, validateResponse) {
+  // Resolve #ref(s) in schema, returns object { spec: THE_RESOLVED_SCHEMA }
+  const resolvedSchema = SwaggerClient.resolve({ spec })
```

Comment:
This is a massive module. I think @ivan-tymoshenko has a thinner/better alternative.

Reply:
I could write a tool to get the `$refs` resolved, waiting for @ivan-tymoshenko though

Reply:
I had to use `@apidevtools/json-schema-ref-parser`